app/lib/analyzer/backend.dart

```diff
@@ -111,19 +118,22 @@ class AnalysisBackend {
         inserts.add(package);
       }
 
-      final DateTime prevTimestamp = version?.analysisTimestamp;
+      final Duration ageDiff = version == null
+          ? null
+          : version.analysisTimestamp.difference(version.analysisTimestamp);
```

Comment:
I think you might have a typo here, since this is basically doing `version.analysisTimestamp - version.analysisTimestamp` which is always `0`.

Reply:
yeah, fixing...